conans/server/rest/controllers/v2/conan_controller.py

```diff
@@ -17,13 +17,13 @@ def attach_to(self, app):
         @app.route(r.package, method=["GET"])
         @app.route(r.package_recipe_revision, method=["GET"])
         @app.route(r.package_revision, method=["GET"])
-        def get_package_snapshot(name, version, username, channel, package_id, auth_user,
-                                 revision=None, p_revision=None):
+        def get_package_file_list(name, version, username, channel, package_id, auth_user,
```

Comment:
Maybe convert it to a dict {file: {} }, to be extensible for the future. Can be {file: None} now, but that it will understand in the future: 
```
{file = {size=12355, md5=012312312, ...},
 file2 = {....}
```